style-guide.rst

```diff
@@ -0,0 +1,628 @@
+*****************
+Style Guide
+*****************
+
+.. _spelling-and-grammar:
```

Comment:
It'd be good to add a section on acronyms. That is, avoid if possible.

Reply:
Agree.

Reply:
I agree with you in theory, but-
I can't think of any examples of *bad* acronyms or initialisms.

We use, all the time:

 - ODK
 - XML
 - adb
 
We should avoid Latin, though:

 - e.g.
 - i.e.
 - etc.